tests/run-docker

```diff
@@ -8,7 +8,16 @@ export LIZMAP_GROUP_ID=${LIZMAP_GROUP_ID:-$(id -g)}
 CMD=$1
 
 if [ "$CMD" == "" ]; then
-    CMD="up -d"
+  CMD="up -d"
+fi
+
+# Install or update Lizmap as a QGIS Server plugin
+if [ "$CMD" != "stop" ]; then
```

Comment:
you should verify first that the directory qgis-server-plugins/lizmap/ does not already exist, else each time we launch the stack, it will have a download. This is not really that we want IMHO

Reply:
You could also do this statement when `$CMD="build"`.

`if [ !- d qgis-server-plugins/lizmap/  -o "$CMD" == "build" ]; then`

Reply:
I did that to always have the last Lizmap plugin version in a simple manner.

Reply:
well, if you want the latest version, you rebuild, and that's all. but you should not install the plugin each time we up the stack